bundle/src/main/java/com/adobe/acs/commons/ondeploy/OnDeployExecutor.java

```diff
@@ -17,7 +17,7 @@
  * limitations under the License.
  * #L%
  */
-package com.adobe.acs.commons.ondeploy.impl;
+package com.adobe.acs.commons.ondeploy;
```

Comment:
Since I wrote the MBean support for this, I feel qualified to comment here.  I love the idea of having a better public API for executing these that fully supports updating status nodes, etc.  But I don't personally think the current internal mbean api makes for a suitable public API.  It is "stringly" typed due to the constraints of JMX, and it exposes the JMX TabularDataSupport object which feels wrong to me.

I would rather leave the Mbean interface internal only (maybe rename it to OnDeployExecutorMBean?), and then create a new interface for public usage, that uses the `OnDeployScript` and `OnDeployScriptProvider` interfaces rather than passing around class names/packages as strings.  More or less it could expose the existing runScript and runScript methods from OnDeployExecutorImpl, but also add support for a force parameter, to force re-execution of already completed scripts.

Will defer to @HitmanInWis and others for any decisions, but just sorta my $.02.

Reply:
Thank you. I agree with the suggested approach.

Reply:
Implemented like suggested. Please verify again

Reply:
👍